diff --git a/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.js b/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.js
index 09791a7..022cb03 100644
--- a/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.js
+++ b/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.js
@@ -1,12 +1,12 @@
 'use strict';
 
-var constants = require('react-reconciler/constants');
+var constants = require('react-reconciler/constants.js');
 var store = require('../store.js');
 var log = require('./log.js');
 
 "use strict";
 function resolveUpdatePriority() {
-  log.log("info", "lifecycle::resolveUpdatePriority");
+  log.log("info", "lifecycle::resolveUpdatePriority.js");
   if (store.store.currentUpdatePriority) {
     return store.store.currentUpdatePriority;
   }
diff --git a/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.mjs b/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.mjs
index d728912..6ab5956 100644
--- a/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.mjs
+++ b/node_modules/@pixi/react/lib/helpers/resolveUpdatePriority.mjs
@@ -1,4 +1,4 @@
-import { DefaultEventPriority, ContinuousEventPriority, DiscreteEventPriority } from 'react-reconciler/constants';
+import { DefaultEventPriority, ContinuousEventPriority, DiscreteEventPriority } from 'react-reconciler/constants.js';
 import { store } from '../store.mjs';
 import { log } from './log.mjs';
 
diff --git a/node_modules/@pixi/react/lib/store.js b/node_modules/@pixi/react/lib/store.js
index 805305c..c556115 100644
--- a/node_modules/@pixi/react/lib/store.js
+++ b/node_modules/@pixi/react/lib/store.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var constants = require('react-reconciler/constants');
+var constants = require('react-reconciler/constants.js');
 
 "use strict";
 const store = {
diff --git a/node_modules/@pixi/react/lib/store.mjs b/node_modules/@pixi/react/lib/store.mjs
index c086900..f09c663 100644
--- a/node_modules/@pixi/react/lib/store.mjs
+++ b/node_modules/@pixi/react/lib/store.mjs
@@ -1,4 +1,4 @@
-import { DefaultEventPriority } from 'react-reconciler/constants';
+import { DefaultEventPriority } from 'react-reconciler/constants.js';
 
 "use strict";
 const store = {
